assets/css/layout/_header.scss

```diff
@@ -0,0 +1,149 @@
+// Header styles
+
+header {
+    .brand {
+        a {
+            color: $black;
+            font-size: rem(48);
+            font-weight: normal;
```

Comment:
@greatislander suggested we use the actual numeric font-weight instead of keywords. For example using `400` instead of `normal`. sass variables for these values are also available in _variable.scss. In this case, you would use `$font-weight-normal` instead of `normal`